### DISCUSSION TITLE
Alguien sabe donde esta la sesión de hoy backed NodeJS?

### DISCUSSION THREAD

**Discussion by Ricardo-g-garcia** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#581] [General]
no me aparece el link, alguien sabe si hubo sesion en vivo y donde puedo checarla?

**Comment by ismagician** [comment]
Aquí puedes ver el live: https://web.microsoftstream.com/video/2de8195c-44bc-40eb-b479-daa19f3bb9da

**Reply by Ricardo-g-garcia** [reply]
Muchas Gracias Ismael

**Reply by ismagician** [reply]
De nada 👍
